crates/tabby-common/src/index/mod.rs

```diff
@@ -64,6 +65,7 @@ impl IndexSchema {
                         .set_fieldnorms(true),
                 ),
         );
+        let field_source = builder.add_text_field("source", STRING | FAST);
```

Comment:
since the field is only meaningful for web docs, add it to `./doc.rs` as a json field, instead of adding it to schema.